Configure

```diff
@@ -43,7 +43,7 @@ my $usage="Usage: Configure [no-<cipher> ...] [enable-<cipher> ...] [-Dxxx] [-lx
 #
 # --cross-compile-prefix Add specified prefix to binutils components.
 #
-# --api         One of 0.9.8, 1.0.0, 1.0.1, 1.0.2, 1.1.0, 1.1.1, or 3.0.0 / 3.
+# --api         One of 0.9.8, 1.0.0, 1.0.1, 1.0.2, 1.1.0, 1.1.1, or 4.0.0
```

Comment:
I removed ``4`` because ``--api=4`` does not work because ``"4"`` is not in ``apitable``. I could also add `` "4" => 4,`` to the API table.